src/main/docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17.0.2_8-jre-focal AS jre-build
+FROM eclipse-temurin:11.0.14.1_1-jre-focal AS jre-build
 
 FROM debian:bullseye-20220125-slim
```

I think we had a misunderstanding regarding the digest pinning as well (wasn't fast enough to respond on that PR for timezone reasons as well... 😅). What I meant was pinning to the SHA256 digest:

```suggestion
FROM eclipse-temurin:11.0.14.1_1-jre-focal@sha256:246f0a07f7ba2c52b48d1879aa6f294d80619575f719ea31eaec0885560b803e AS jre-build

FROM debian:bullseye-20220125-slim@sha256:4c25ffa6ef572cf0d57da8c634769a08ae94529f7de5be5587ec8ce7b9b50f9c
```

See for example https://docs.renovatebot.com/docker/#digest-pinning for more details.

---

Yeah, it'd be great if this would be a little more obvious. There are numerous open issues to indicate this stuff in DockerHub, the Docker CLI and so on, but they've been open for *years*... 😅

---

I am more impressed that this also still not mentioned anywhere in the OCI standard space, but hey you never stop learning so thank you again for this heads up :smiley:

---

This is one of those pull requests that reminds me that everyday is a school day. ;-) Thanks for the collaboration everyone. Much appreciated.